### PR TITLE
#154 - test for FilePackage

### PR DESCRIPTION
**Pull request by olenagerasimova**
Created `FilePackageTest` for #154.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/rpm-adapter/issues/305#issuecomment-651583345))

@olenagerasimova Job `gh:artipie/rpm-adapter#305` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/rpm-adapter/issues/305#issuecomment-651583345))

@olenagerasimova Pull request [#305](https://github.com/artipie/rpm-adapter/issues/305) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

File: src/main/java/com/artipie/rpm/pkg/PackageOutput.java [hunk 1]
```diff
@@ -91,13 +91,19 @@
              */
             private final Path file;
 
+            /**
+             * Was package output was accepted?
```
Comment (olegmoz) [inline src/main/java/com/artipie/rpm/pkg/PackageOutput.java:95]: @olenagerasimova `was` duplicated in the sentence

File: src/test/java/com/artipie/rpm/pkg/FilePackageTest.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    void returnsPath() {
+        final Path path = Paths.get("some/path/package.rpm");
+        MatcherAssert.assertThat(
+            new FilePackage(path).path(),
+            new IsEqual<>(path)
+        );
+    }
+
+    @Test
+    void returnsParsedFile() throws IOException {
```
Comment (olegmoz) [inline src/test/java/com/artipie/rpm/pkg/FilePackageTest.java:53]: @olenagerasimova please consider adding negative test, when parsing fails

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected, please, take a look

**Comment by 0crat** [comment]
Job `gh:artipie/rpm-adapter#305` is not assigned, can't get performer

**Pull request merged**